alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/alerts/AlertIndices.kt

```diff
@@ -184,6 +192,16 @@ class AlertIndices(
         }
     }
 
+    private fun updateIndexMapping(index: String, indexPrefix: Boolean = false) {
+        val clusterState = clusterService.state()
+        when {
+            indexPrefix -> clusterState.metaData.indices.keysIt().forEach {
+                key -> if (key.startsWith(index)) SchemaVersionUtils.updateIndexMapping(key, MAPPING_TYPE, alertMapping(),
```

Comment:
I assume this is for updating all the history indexes? Is this needed or will updating the mappings for `*alerts*` update all of them in one go?

Reply:
Update all alerts history index mapping to keep consistent. It's easier for user to use same mapping rather than different versions.
Will check the schema version and update the mapping if it's not the latest one. This will only happen once for each index.